hwtracer/Cargo.toml

```diff
@@ -24,3 +24,6 @@ iced-x86 = { version = "1.18.0", features = ["decoder"]}
 [build-dependencies]
 cc = "1.0.62"
 rerun_except = "0.1.2"
+
+[features]
+yk_testing = []
```

Comment:
Should we add a comment here along the lines of "internal flag, used only for testing"? Otherwise external users might wonder if they should care about it.

Reply:
We could do, but this idiom appears in many crates in the workspace.

I'm about to do a ykdocs update, so maybe it would be better said there just once?

Reply:
Works for me!